addons/message_try_firefox_extension/manifest.json

```diff
@@ -0,0 +1,60 @@
+{
+  "api_version": "0.1",
+  "id": "message_try_firefox_extension",
+  "name": "Try the Firefox extension",
+  "type": "message",
+  "conditions": {
+    "min_client_version": "2.25.1",
+    "enabled_features": ["webExtension"],
+    "trigger_time": 1209600,
+    "platforms": ["windows"]
+  },
+  "message": {
+    "date": 1733157651,
+    "usesSharedStrings": true,
+    "shortVersion": "N/A",
```

Comment:
should this be "" ?

Reply:
Updated this. It doesn't matter, as it's never shown - it's used for update/what's new messages (to drop a version number in the "Update to ___" slot). 

Potentially this was a bad design choice of mine to always require it (messages using shared strings don't pass validation without it). I'm not inclined to update it right now, given how rarely we have messages like this - though let me know if you think it's needed now.